backend/molgenis-emx2/src/main/java/org/molgenis/emx2/ColumnType.java

```diff
@@ -31,6 +31,7 @@ public enum ColumnType {
   DATE_ARRAY(LocalDate[].class, ORDINAL_OPERATORS),
   DATETIME(LocalDateTime.class, ORDINAL_OPERATORS),
   DATETIME_ARRAY(LocalDateTime[].class, ORDINAL_OPERATORS),
+  DURATION(Integer.class, ORDINAL_OPERATORS),
```

Comment:
I think this is a format flavor? I.e. you should use the pattern you can see on line 48-54. I.e. then you add this type using `DURATION(INTEGER)`